Project-I-Social-Card/social-card/src/components/CardComponents/Card.css

```diff
@@ -1 +1,21 @@
 /* All styles for your Card Components live here */
+
+.card-container{
+    display: flex;
+    flex-direction: column;
+    /* justify-content: flex-end; */
+    align-items: flex-end; 
+}
+#react-img{
+    width: 88%;
+}
+.card-content{
+    width: 88%;
+}
```

Comment:
The `.card-content` component also has a gray rounded `border` at its right, bottom and left sides.